### PR TITLE
Increase PHP version in CI to 7.3

### PR DESCRIPTION
**Pull request by Jean85**

### CODE REVIEW

**Comment by Jean85** [comment]
CI seems to be turned off :cry: 

https://travis-ci.org/ekino/EkinoNewRelicBundle/requests

**Comment by jderusse** [comment]
@Nyholm to you have the required permissions to re-enable the automated build?

**Comment by Nyholm** [comment]
No, I dont. I think you and I has equal access rights.

Could @rande help us activate Travis?
https://travis-ci.org/ekino/EkinoNewRelicBundle

**Comment by jderusse** [comment]
can you please rebase the PR ?

**Comment by Jean85** [comment]
Done.

File: .travis.yml [hunk 1]
```diff
@@ -15,25 +15,25 @@ matrix:
...
           # Minimum supported dependencies with the latest and oldest PHP version
-        - php: 7.2
+        - php: 7.3
           env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" SYMFONY_DEPRECATIONS_HELPER="weak_vendors"
         - php: 7.1
           env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" SYMFONY_DEPRECATIONS_HELPER="weak_vendors"
 
           # Test the latest stable release
         - php: 7.1
-        - php: 7.2
```
Comment (Nyholm) [inline .travis.yml]: We should not remove 7.2. Right?

Comment (Jean85) [reply .travis.yml]: Right. I though we still got it from the matrix, but we don't. Fixing now.

**Pull request merged**